Spigot-API-Patches/0005-Adventure.patch

```diff
@@ -20,7 +20,7 @@ index e3744ee042e0426a513c4fdf4abedafe85e31cd2..75b2830340051deb0fa39149e80872d2
 +            <dependency>
 +                <groupId>net.kyori</groupId>
 +                <artifactId>adventure-bom</artifactId>
-+                <version>4.7.0</version> <!-- keep this version in synch with javadoc section below! -->
++                <version>4.7.0</version> <!-- keep this version in sync with javadoc section below! When updating this make sure to update the linked JavaDocs on the homepage as well!-->
```

Comment:
```suggestion
+                <version>4.7.0</version> <!-- keep this version in synch with javadoc section below *and* on the papermc.io repository! -->
```

Reply:
:( the extra h makes me unhappy

though -- can you extract this into a maven project property so there's only one place in the pom that needs updating? I think that should work in javadoc `links` URLs too, but definitely works in pom versions.

Reply:
Done